#test exception 8
{
    a = 1;
    b = 2/(a-1);
    return a+b;
}
